.travis.yml

```diff
@@ -15,22 +15,20 @@ before_install:
   - pip install pytest-cov
   - pip install coveralls
   - pip install future
+  # Install version specific packages for 2.7 and 3.5
+  - pip install 'pandas<0.25'
+  - pip install xarray
   - pip install matplotlib
-  - pip install pysatCDF >/dev/null
   - pip install apexpy
-
-  # Prepare modified pysat install
   - pip install numpy
-  - cd ..
```

Comment:
This line shouldn't be deleted.  Need to go up one level before installing via git

Reply:
This should be after the pip install pysaCDF line